src/app/COMPONENTS/create-task/create-task.component.html

```diff
@@ -22,6 +22,16 @@ <h1>Create Task</h1>
             </mat-select>
             <mat-error>Type is required</mat-error>
         </mat-form-field>
+
+        <mat-form-field>
+            <mat-label>Assigned To</mat-label>
+            <mat-select formControlName="assignedTo" placeholder="Asiggned to">
+                <mat-option value="UNASSIGNED"> Unassigned </mat-option>
+                <mat-option value={{user.id}} *ngFor="let user of users$ | async">{{user.firstName}} {{user.lastName}} - {{user.username}}</mat-option>
+            </mat-select>
+            <mat-error>Type is required</mat-error>
```

Comment:
you don't need a mat error here, do you?

Reply:
Yup :)